agent/command/test_selection_get.go

```diff
@@ -92,7 +110,20 @@ func (c *testSelectionGet) Execute(ctx context.Context, comm client.Communicator
 		c.OutputFile = GetWorkingDirectory(conf, c.OutputFile)
 	}
 
-	if !c.isTestSelectionAllowed(conf) {
+	calledAPI := false
+	defer trace.SpanFromContext(ctx).SetAttributes(attribute.Bool(testSelectionCalledAttribute, calledAPI))
+
+	enabled := c.isTestSelectionAllowed(conf)
+	trace.SpanFromContext(ctx).SetAttributes(attribute.Bool(testSelectionEnabledAttribute, enabled))
+	trace.SpanFromContext(ctx).SetAttributes(attribute.StringSlice(testSelectionInputTestsAttribute, c.Tests))
+	if c.Strategies != "" {
+		trace.SpanFromContext(ctx).SetAttributes(attribute.String(testSelectionStrategiesAttribute, c.Strategies))
```

Comment:
Would probably be a bit cleaner to make this attribute a string slice rather than the string because if the user is looking for a particular strategy, it'll be harder to parse from a string of multiple ones.